### DISCUSSION TITLE
A script for easy integration with SteamPlay

### DISCUSSION THREAD

**Discussion by pan-mroku** [EDCD/EDMarketConnector#837] [Show and tell]
Here's a script that can be included in Steam launch options, so EDMarketConnector is ran automatically with the game. It is advisable to close it manually when exiting the game, so it closes cleanly.
```
#!/bin/bash

VIRTUALENV_DIR="venv"

DIR=`dirname "${BASH_SOURCE[0]}"`
cd $DIR

if [ -e "$VIRTUALENV_DIR"/bin/activate ]; then
		source ./"$VIRTUALENV_DIR"/bin/activate
fi

./EDMarketConnector.py > log_`date +"%Y_%m_%d_%H%M"` 2>&1 &
disown
cd $OLDPWD
```

Save it as for instance `venvandlog.sh` in your EDMarketConnector directory, enable execution permission `chmod +x venvandlog.sh` and put `<path to your EDMarketConnector directory>/venvandlog.sh; %command%` into steam launch options of the game.

**Comment by jeffstokes72** [comment]
hi @pan-mroku is edmcoverlay2 dead? I can't get it to work and it looked like the last branch for gtk support was never merged into your main. I forked your repo, merged the gtk changes and tested, doesn't seem to overlay anything that I can see though it says it works. Sorry for necroing this, I couldn't figure out how to reach you based off your git profile.

**Reply by pan-mroku** [reply]
Turns out on GitHub I had to enable issues in settings for them to appear 
Let's move to https://github.com/pan-mroku/edmcoverlay2/issues

Main branch should be working, I use it regularly
